OtripleS.Web.Api/Brokers/Storage/StorageBroker.Fees.cs

```diff
@@ -0,0 +1,51 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using OtripleS.Web.Api.Models.Fees;
+
+namespace OtripleS.Web.Api.Brokers.Storage
+{
+    public partial class StorageBroker
+    {
+        public DbSet<Fee> Fees { get; set; }
+
+        public async ValueTask<Fee> InsertFeeAsync(Fee fee)
+        {
+            var feeEntityEntry = await this.Fees.AddAsync(fee);
```

Comment:
If the right side of your code isn't clear about the type, please don't use `var` instead use the explicit type of the entity, in this case it would be `EntityEntry<Fee> fee = .... `

Reply:
Do the same for the rest please